common/addressing/defaults.go

```diff
@@ -32,7 +32,7 @@ const (
 	// Default IPv4 prefix length of entire cluster
 	DefaultIPv4ClusterPrefixLen = 8
 	// Default IPv6 prefix to represent NATed IPv4 addresses
-	DefaultNAT46Prefix = "aa46::/48"
+	DefaultNAT46Prefix = "f00d::/48"
```

Comment:
```
        v6.saddr.in6_u.u6_addr32[0] = v6prefix_src->p1;
        v6.saddr.in6_u.u6_addr32[1] = v6prefix_src->p2;
        v6.saddr.in6_u.u6_addr32[2] = v6prefix_src->p3;
        v6.saddr.in6_u.u6_addr32[3] = v4.saddr;
```

This will only work by accident I think. By using the IPv4 as the trailing 4 bytes, you are setting endpoint id, the state bits and some of the node id. This may conflict with a properly allocated IP.

I think we need a NAT46prefix which contains the node id and is distinguishable as a NAT46 address.